integration/container/mounts_linux_test.go

```diff
@@ -206,3 +209,91 @@ func TestMountDaemonRoot(t *testing.T) {
 		})
 	}
 }
+
+func TestTmpfsMounts(t *testing.T) {
```

Comment:
Same here -- I would drop this test case, I don't see how it can bring some additional value.

If you don't want to drop it, at least use `container.Exec()` from `integration/internal/container/exec.go`, it will greatly simplify the code.

Reply:
Thank you, removed.